entrypoint/versionbump/coredns.go

```diff
@@ -0,0 +1,63 @@
+//go:build coredns
+
+package versionbump
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/pkg/template"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("VersionTemplate Upgrade:", func() {
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus())
+	})
+
+	It("Verifies bump version for coredns on rke2", func() {
```

Comment:
Do we need to add a check for product? This test should not be executed for k3s, right?

Reply:
rke2 only

Reply:
you mean like break if it is k3s ? 

this we will add later on when improved the main entrypoint , right now all version bumps that are for a single product we cant do much expect 

`if cfg.product == {product} { os.exit()} ` 

but thats not a good option

Reply:
task related 
https://github.com/rancher/qa-tasks/issues/803